src/components/20-molecules/accordion/README.md

```diff
@@ -23,3 +23,11 @@ String-valued `title` sets the title of the accordion.
 ### icon
 
 String-valued `icon` sets the additional icon of the accordion according to a valid `<svg>...</svg>` value. Leave unset if no icon is required.
+
+### onStateChange
+
+The function-valued attribute `onStateChange` can be used as a callback prop for React and other frameworks. The value(`isOpen`) is the `open` property.
```

Comment:
Rewrite the 2nd sentence:

The passed-in function gets called whenever the accordion's state changes, and it receives a Boolean parameter that mirrors the `open` property value.